CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.16
```

Comment:
Given that we made the feature GA, I would support making plugin stable 1.0 version.